src/devices/Common/Iot/Device/Common/WeatherHelper.cs

```diff
@@ -12,6 +12,21 @@ namespace Iot.Device.Common
     /// </summary>
     public static class WeatherHelper
     {
+        /// <summary>
+        /// Gas constant of dry Air, J / (kg * K)
+        /// </summary>
+        public const double SpecificGasConstantOfAir = 287.058;
+
+        /// <summary>
+        /// Gas constant of vapor, J / (kg * K)
+        /// </summary>
+        public const double SpecificGasConstantOfVapor = 461.523;
+
+        /// <summary>
+        /// Default atmospheric temperature gradient = 0.0065K/m (or 0.65K per 100m)
+        /// </summary>
+        public const double DefaultTemperatureGradient = 0.0065;
```

Comment:
I'd recommend to keep them private/internal, if we make it public let's at least expose UnitsNet

Reply:
There's likely no Unitsnet equivalent for this strange unit.